plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -558,7 +563,7 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
             assignVariables.add(it)
             addToCache(leftOp, it)
         }
-        projectOp(rightOp, 1)?.let {
+        projectOp(rightOp, 2)?.let {
```

Comment:
This is failing a test under `testing/src/test/scala/..../querying/CallTests::"should contain a call node for 'add' with correct fields"` but this is because the test is expecting `order` to be 1. You can simply update the test in this case.

Reply:
> Is this linked to #133 ?

Yes but, I think that needs more fixes.